src/Form/FenceReviewCaseType.php

```diff
@@ -49,6 +50,10 @@ public function buildForm(FormBuilderInterface $builder, array $options)
                     'data-specifier' => 'bringer',
                 ],
             ])
+            ->add('bringerIsUnderAddressProtection', CheckboxType::class, [
+                'label' => '!'.$this->translator->trans('Is under address protection', [], 'case').'!',
```

Comment:
See previous comment on “!” and translatable text.

Reply:
Updated.